enhancement_worker_templates/web/typescript_express/package.json

```diff
@@ -11,7 +11,7 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "typecheck": "echo Typechecking... && tsc --pretty"
   },
-  "author": "The Paranoids",
+  "author": "ASHIRT Ops",
```

Comment:
do we want to change the author as well? or just the paths

Reply:
Let's just make it "ASHIRT Authors"

Reply:
sounds good  - changed it